code/modules/mob/language/outsider.dm

```diff
@@ -115,3 +115,17 @@
 		return 1
 
 	return 0
+
+//for your antag purposes. machine-translatable, if you don't want that use Cult or one of the animal langs
```

Comment:
Machine-translatable? Not sure that's a feature?

Reply:
has machine_can_understand = 1 like most languages, so the translator still works on it.

Reply:
Yeah, if this is admin use only I'm not certain that's desirable.

Reply:
I don't really care, I don't see myself using it, though.